hips/tiles/hipsdescription.py

```diff
@@ -0,0 +1,75 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""HiPS description class -- provides methods for parsing the HiPS file properties."""
+
+__all__ = [
+    'HipsDescription',
+]
+
+from collections import OrderedDict
+
+
+class HipsDescription:
+    """This class provides methods for parsing the HiPS properties file.
+       It also provides multiple getters around the commonly used properties.
+    """
+
+    def __init__(self, properties: str) -> None:
+        super(HipsDescription, self).__init__()
+        self.properties = self.parse_file_properties(properties)
+
+    @classmethod
+    def parse_file_properties(cls, properties: str) -> OrderedDict:
+        """Returns an ordered dictionary containing the HiPS file properties.
+
+        This method parses the HiPS file properties and loads it in a dictionary,
+        then is casts it to `collections.OrderedDict` type.
+
+        Parameters
+        ----------
+        properties : str
+            HiPS file properties
+        
+        Returns
+        -------
+        dict_properties : OrderedDict
+        """
+        properties = properties.split('\n')
+        dict_properties = {}
+        for property in properties:
+            key_value = property.split('=')
+            try:
+                dict_properties[key_value[0].strip()] = key_value[1].strip()
+            except:
+                pass
+        return OrderedDict(dict_properties)
+
+    def get_properties(self) -> OrderedDict:
+        """OrderedDict: Returns the ordered dictionary containing the HiPS properties."""
+        return self.properties
+
+    def get_base_url(self) -> str:
```

Comment:
I think you should change all of these
```
def get_something(self):
    return some_type(self.properties['something'])
```
to
```
@property
def something(self):
    return some_type(self.properties['something'])
```
i.e. use Python properties instead of getter methods, when there are no arguments to the getter.

Also, is it useful  to try-except and return none if the info isn't there?
If you don't have the try-except you'll get a KeyError and that's a bit better no?
I mean: if a caller doesn't handle the case of a non-existent key, then getting an error at that point is better than getting none and failing later.
and if a caller can handle the case of a non-existent key, she can just as easy use try-except KeyError and decide what to put, as it would be to decide what to do with the None return value.